tests/datasets/test_forms.py

```diff
@@ -0,0 +1,102 @@
+from datetime import datetime
+import pytest
+from django.urls import reverse
+from django_webtest import DjangoTestApp
+from vitrina.datasets.factories import DatasetFactory
+from vitrina.datasets.models import Dataset
+from vitrina.orgs.factories import OrganizationFactory
+from vitrina.users.models import User
+
+
+@pytest.mark.django_db
+def test_change_form_no_login(app: DjangoTestApp):
+    response = app.get(reverse('dataset-change', kwargs={'org_kind': 'test_org_kind',
+                                                         'org_slug': 'test_org_slug',
+                                                         'slug': 'test_dataset_slug'}))
+    assert response.status_code == 302
+    assert response.location == '/dataset/test_org_slug/'
+
+
+@pytest.mark.django_db
+def test_change_form_wrong_login(app: DjangoTestApp):
+    user = User.objects.create_user(email="test@test.com", password="test123")
+    app.set_user(user)
+    response = app.get(reverse('dataset-change', kwargs={'org_kind': 'test_org_kind',
+                                                         'org_slug': 'test_org_slug',
+                                                         'slug': 'test_dataset_slug'}))
+    assert response.status_code == 302
+    assert response.location == '/dataset/test_org_slug/'
+
+
+@pytest.mark.django_db
+def test_change_form_correct_login(app: DjangoTestApp):
+    org = OrganizationFactory(
+        title="Org_title",
+        created=datetime(2022, 8, 22, 10, 30),
+        jurisdiction="Jurisdiction1",
+        slug='test-org-slug',
+        kind='test_org_kind'
+    )
```

Comment:
Organization should be created automatically by `DatasetFactory`.

https://factoryboy.readthedocs.io/en/stable/reference.html?highlight=SubFactory#subfactory